funnel/templates/siteadmin_layout.html.jinja2

```diff
@@ -0,0 +1,30 @@
+{% extends "layout.html.jinja2" %}
+{% from "baseframe/forms.html.jinja2" import renderform, rendersubmit %}
+
+{% block top_title %}
+  <h1 class="mui--text-display1"><strong>Site admin</strong></h1>
+{% endblock %}
+
+{% block contentwrapper -%}
+  <div class="grid">
+    {# <div class="grid__col-xs-12 grid__col-sm-5 grid__col-md-4 grid__col-lg-3">
+      <div class="sticky-admin-panel">
+        <div class="card">
+          <div class="card__header">
+            <h3 class="mui--text-title">{% trans %}Moderation sections{% endtrans %}</h3>
+          </div>
+          <div class="card__body">
+            <ul class="mui-list--aligned mui-list--unstyled">
+              <li><p><a href="{{ url_for('siteadmin_comments') }}">{% trans %}Comments{% endtrans %}</a></p></li>
+            </ul>
+          </div>
+        </div>
+      </div>
+    </div> #}
+
+    <div class="grid__col-xs-12 grid__col-sm-12 grid__col-md-12 grid__col-lg-12">
+      {% block admincontentblock %}
+      {% endblock %}
+    </div>
+  </div>
+{%- endblock %}
```

Comment:
This page doesn't seem to have a route

Reply:
It's a base template. All the other siteadmin templates will extend from it.